pages/resultset.py

```diff
@@ -17,21 +17,27 @@ class ResultsetPage(Base):
     _logviewer_button_locator = (By.ID, 'logviewer-btn')
     _resultset_locator = (By.CSS_SELECTOR, 'div.row.result-set')
     _result_status_locator = (By.ID, 'job-details-panel')
+    _unclassified_failure_count_locator = (By.CSS_SELECTOR, '.btn-unclassified-failures')
 
     @property
     def job_result_status(self):
         return self.selenium.find_element(*self._job_result_status_locator).text
 
+    @property
+    def verify_unclassified_failure_exists(self):
+        return self.selenium.find_element(*self.unclassified_failure_count_locator).text
```

Comment:
Can we test this span to see if its value > 0 ? I could put an id on it to allow more robust selection if you like. Aiui it only updates after the DOM is populated, it's always a number (that could be converted from string if you wish in the test harness), and if the count button ever broke we'd indirectly be testing that also.
![countspan](https://cloud.githubusercontent.com/assets/3660661/9305779/0592fa84-44c2-11e5-9072-b5eb9eb4a663.jpg)